packages/core/ui/Snackbar.tsx

```diff
@@ -57,19 +57,36 @@ function MessageSnackbar({
     popSnackbarMessage()
     setOpen(false)
   }
-
-  const [message, level] = snackbarMessage || []
+  const [message, level, retry] = snackbarMessage || []
   return (
-    <Snackbar
-      open={open && !!message}
-      onClose={handleClose}
-      action={
-        <IconButton aria-label="close" color="inherit" onClick={handleClose}>
-          <CloseIcon />
-        </IconButton>
-      }
-    >
-      <Alert onClose={handleClose} severity={level || 'warning'}>
+    <Snackbar open={open && !!message} onClose={handleClose}>
+      <Alert
+        onClose={handleClose}
+        action={
+          retry ? (
+            <>
+              <Button
+                color="inherit"
+                variant="contained"
```

Comment:
I think the default styling (i.e. getting rid of `variant="contained"`) is more typical for snackbars.